owncloudApp/src/main/java/com/owncloud/android/ui/adapter/FileListListAdapter.java

```diff
@@ -202,15 +207,25 @@ public View getView(int position, View convertView, ViewGroup parent) {
                     TextView fileSizeV = view.findViewById(R.id.file_size);
                     TextView fileSizeSeparatorV = view.findViewById(R.id.file_separator);
                     TextView lastModV = view.findViewById(R.id.last_mod);
-
-                    lastModV.setVisibility(View.VISIBLE);
-                    lastModV.setText(DisplayUtils.getRelativeTimestamp(mContext, file.getModificationTimestamp()));
-
-                    fileSizeSeparatorV.setVisibility(View.VISIBLE);
-                    fileSizeV.setVisibility(View.VISIBLE);
-                    fileSizeV.setText(DisplayUtils.bytesToHumanReadable(
-                            file.getFileLength(), mContext
-                    ));
+                    
+                    if(!mOnlyAvailableOffline){
+                        lastModV.setVisibility(View.VISIBLE);
+                        lastModV.setText(DisplayUtils.getRelativeTimestamp(mContext, file.getModificationTimestamp()));
+
+                        fileSizeSeparatorV.setVisibility(View.VISIBLE);
+                        fileSizeV.setVisibility(View.VISIBLE);
+                        fileSizeV.setText(DisplayUtils.bytesToHumanReadable(
+                                file.getFileLength(), mContext
+                        ));
+                    }else{
+                        lastModV.setVisibility(View.VISIBLE);
```

Comment:
This call is present when mOnlyAvailableOffline is both true and false so can be move out of the if, else. Check the other calls just in case.